Jenkinsfile

```diff
@@ -292,6 +292,12 @@ pipeline {
                         buildHipClangJob('/opt/rocm/llvm/bin/clang++', '', "",  image+'-hip-clang', "/usr/local", cmd)
                     }
                 }
+                stage('Hip release Link DBs') {
+                    agent{label rocmnode("vega20")} 
+                    steps{
+                        buildJob('hcc', '-DMIOPEN_LINK_DB=gfx906_60', '', image+'-link-db', "/opt/rocm", "", "-e MIOPEN_FIND_MODE=Hybrid")
```

Comment:
Why are we passing in an environment variable?

Reply:
MIOpen does not read the system find db in `Normal` find mode by default. The env variable overrides that behavior.

Reply:
Will it fail if embedded system-find-db is not working (i.e. unavailable) due to some reason? I guess no.

I recommend creating a new test, `test_system_find_db`, that should always fail is system-find-db is not available, in test/CMakeLists.txt.
1. Select some (dozen of?) configs that always present in the system-find-db
2. Disable GEMM via environment. Or, better, add a new var that should disable Immediate mode fallback (this is future-proof, I can make it quickly).
3. Make a new mode for `test_immed_conv` that should have enabled system-find-db and disabled user-find-db. Unlike current version, it shouldn't use Find() API calls that populate the user-find-db prior Immediate mode calls. In this mode, test would fail if a config under test does not reside in the system-find-db.

This test would be fast and thus can be included everywhere.

We (me and Vasya) can help with steps 2 and 3.

@DrizztDoUrden @JehandadKhan @pfultz2 What do you think?

Reply:
I think that would be useful. I am not sure if we ever test system FDB now.

Reply:
I am working on this and will update the PR once finished.

Reply:
@atamazov  #273 implements serial 2 above

Reply:
We have item but don't have 1 and 3. Please let us know if you need assistance or if you would like to postpone this.